spec/requests/omniauth_callbacks_request_spec.rb

```diff
@@ -57,11 +57,6 @@
       it "does not create user" do
         expect { do_request }.not_to(change(User, :count))
       end
```

Comment:
I decided to remove this test because the service did not and does not have anything that dealt with an existing user with an invalid uid. But I may have missed something in this analysis. what do you think?

Reply:
and truth! I'm very stupid. there was a wrong detail in the find_or_create_user method and in checking the result in the controller. 

thx!